### ISSUE TITLE
🐛 Handle bad profiles

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/amigos#51]
# Summary

If a user imports a bad profile, it errors forever locks up the application for this tenant. They have no way to work around to unblock themselves. We need to protect the application from saving a bad metadata profile. 

![Image](https://github.com/scientist-softserv/amigos/assets/10081604/9a1b4d61-31f0-4689-915d-f01a365b5cb5)

**Comment by ShanaLMoore** [comment]
Rescue error,
don't crash app,
display error + prevent anything from getting created

consider making a validator in a separate ticket. this is just about failing gracefully.
